todoman/model.py

```diff
@@ -585,8 +586,11 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             q = ', '.join(['?'] * len(lists))
             extra_where.append('AND files.list_name IN ({})'.format(q))
             params.extend(lists)
-        if urgent:
-            extra_where.append('AND priority = 9')
+        if priority:
+            if priority == 'medium' or priority == '!low':
+                extra_where.append('AND priority = 5')
+            elif priority == 'high':
+                extra_where.append('AND priority = 1')
```

Comment:
`high` means anything between `1-4`, so you'll have to write `AND priority <= 4 AND priority > 0`.

For `medium` you can then use `AND priority <= 5 AND priority > 0`

For `low` you can use `AND priority <= 9 AND priority > 0`.